### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mkluwe**
edited link to the `gsl` include directory

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ All tests should pass - indicating your platform is fully supported and you are
 ## Using the libraries
 As the types are entirely implemented inline in headers, there are no linking requirements.
 
-You can copy the [gsl](./gsl) directory into your source tree so it is available
+You can copy the [gsl](./include/gsl) directory into your source tree so it is available
 to your compiler, then include the appropriate headers in your program.
 
 Alternatively set your compiler's *include path* flag to point to the GSL development folder (`c:\GSL` in the example above) or installation folder (after running the install). Eg.
```
Comment (CaseyCarter) [inline README.md]: This line needs to be changed as well, e.g. "...set your compiler's *include path* flag to point to the GSL include folder (`c:\GSL\include` in the example above) or ..."

Comment (mkluwe) [inline README.md]: I've updated my edit on mkluwe:master

**Comment by CaseyCarter** [comment]
+1 from me: Thanks for correcting my failure to update the documentation ;)

**Comment by neilmacintosh** [comment]
This looks good to me, thanks! @mkluwe and thanks @CaseyCarter for picking up the last of the details to be updated.

**Pull request merged**